.github/workflows/skywalking.yaml

```diff
@@ -404,6 +404,13 @@ jobs:
           - name: Alarm BanyanDB
             config: test/e2e-v2/cases/alarm/banyandb/e2e.yaml
 
+          - name: Baseline-driven Alarm ES
+            config: test/e2e-v2/cases/baseline/es/e2e.yaml
+          - name: Baseline-driven Alarm ES Sharding
+            config: test/e2e-v2/cases/baseline/es/es-sharding/e2e.yaml
+          - name: Baseline-driven Alarm BanyanDB
+            config: test/e2e-v2/cases/baseline/banyandb/e2e.yaml
```

Comment:
@wankai123 @kezhenxu94 According to discussion between Han and me, JDBC storage is not suitable for this, because the baseline is calculated based on historic data, which can't be inserted as JDBC tables are created for today only, can't be auto-created between TTL to Now. We have to skip that case.

But, logically, this process does not rely on storage implementation, we just read historic data through GraphQL, hence, we believe we should be good.